easybuild/easyconfigs/p/PyTorch/PyTorch-1.11.0-foss-2021b-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,151 @@
+    # PyTorch-1.10.0_fix-test-dataloader-fixed-affinity.patch
+    '313dca681f45ce3bc7c4557fdcdcbe0b77216d2c708fa30a2ec0e22c44876707',
+    # PyTorch-1.10.0_fix-test-model_dump.patch
+    '339148ae1a028cda6e750ac93fa38a599f66c7abe26586c9219f1a206ea14557',
+    # PyTorch-1.10.0_fix-vsx-vector-functions.patch
+    '7bef5f96cb83b2d655d2f76dd7468a171d446f0b3e06da2232ec7f886484d312',
+    # PyTorch-1.10.0_fix-XNNPACK-tests.patch
+    'd3e749a2a42efce463e3b8a1aebb21f0edf2256682c4417297d9a44a6210e5f8',
+    'ac05943bb205623f91ef140aa00869efc5fe844184bd666bebf5405808610448',  # PyTorch-1.10.0_skip_cmake_rpath.patch
+    # PyTorch-1.10.0_skip-nnapi-test-without-qnnpack.patch
+    '34ba476a7bcddec323bf9eca083cb4623d0f569d081aa3add3769c24f22849d2',
+    # PyTorch-1.11.0_disable_failing_jit_cuda_fuser_tests.patch
+    'e7bfe120a8b3fe2b40dac6839852a5fbab3cb3429fbe44a0fc3a1800adaaee51',
+    # PyTorch-1.11.0_fix-attention_cpp-compilation.patch
+    '84214fcc7e30cf70659a7c3bd70bf11e73d58fd4f7fff2c233e3225619b0e42c',
+    'bb1c4e6d6fd4b0cf57ff8b824c797331b533bb1ffc63f5db0bae3aee10c3dc13',  # PyTorch-1.11.0_fix-fsdp-fp16-test.patch
+    '9a04f4285b800dad8a00c3014af0a9713d40d5dd35d10931c7c0da4e89c558e9',  # PyTorch-1.11.0_fix_sharded_imports.patch
+    '21fc678febcdfbb9dabd72235be23cd392044e9a954f6580d15b530e1f69dcc1',  # PyTorch-1.11.0_fix_skip_jit_cuda_fuser.patch
+    '4f7e25c4e2eb7094f92607df74488c6a4a35849fabf05fcf6c3655fa3f44a861',  # PyTorch-1.11.0_fix-test_utils.patch
+    # PyTorch-1.11.0_increase_c10d_gloo_timeout.patch
+    '20cd4a8663f74ab326fdb032b926bf5c7e94d9750c515ab9050927ba00cf1953',
+    # PyTorch-1.11.0_increase-distributed-test-timeout.patch
+    '087ad20163a1291773ae3457569b80523080eb3731e210946459b2333a919f3f',
+    # PyTorch-1.11.0_increase_test_tolerances_TF32.patch
+    '26e179a4f6f57e49209092612ae5f5cd8c03fd2ca84566ba0244eabefc3736ba',
+    # PyTorch-1.11.0_increase-tolerance-test_ops.patch
+    'ceec745c68a405bba79efb4dc61c662ca84eb950cd0163c7104330f4bf614cf5',
+    'f2e6b9625733d9a471bb75e1ea20e28814cf1380b4f9089aa838ee35ddecf07d',  # PyTorch-1.11.0_install-vsx-vec-headers.patch
+    '8eaca92d64fcadb0552d28e9c7ea5c4bc669d2fe33004e45a3519ce8d0d136a2',  # PyTorch-1.11.0_skip_failing_ops_tests.patch
+    # PyTorch-1.11.1_skip-test_init_from_local_shards.patch
+    '4aeb1b0bc863d4801b0095cbce69f8794066748f0df27c6aaaf729c5ecba04b7',
+    # PyTorch-1.11.1_skip-test_sibling_fusion.patch
+    '3d6f6395d98e8e4ad76b0b63c625fddf082cf7f066eb97d4d82401f96dab2555',
+]
+
+osdependencies = [OS_PKG_IBVERBS_DEV]
+
+builddependencies = [
+    ('CMake', '3.22.1'),
+    ('hypothesis', '6.14.6'),
+]
+
+dependencies = [
+    ('CUDA', '11.4.1', '', SYSTEM),
+    ('Ninja', '1.10.2'),  # Required for JIT compilation of C++ extensions
+    ('Python', '3.9.6'),
+    ('protobuf', '3.17.3'),
+    ('protobuf-python', '3.17.3'),
+    ('pybind11', '2.7.1'),
+    ('SciPy-bundle', '2021.10'),
+    ('typing-extensions', '3.10.0.2'),
```

Comment:
We seemingly bundle this with Python now. Or was there a reason adding this?

Reply:
Ah, good point, there should be no need for it

Reply:
fixed in 8b58193